### PR TITLE
Feature/adjust board component scroll

### PR DESCRIPTION
**Pull request by harshilsharma63**
This PR is a continuation of another PR - https://github.com/mattermost/focalboard/pull/325 . This contains the other PR's changes along with the fixes for cypress tests and a few more enhancements.

I've made two improvements over the work already done by @signalwerk 

The table header in table view and swimlane header in Kannan board also stay in view now when scrolling.

**Kanban View**
kanban

**Table View**
table

cc @signalwerk

### CODE REVIEW

File: webapp/src/components/centerPanel.tsx [hunk 1]
```diff
@@ -129,54 +129,66 @@ class CenterPanel extends React.Component<Props, State> {
...
+                {/*<ViewHeader*/}
+                {/*    boardTree={boardTree}*/}
+                {/*    showView={showView}*/}
+                {/*    setSearchText={this.props.setSearchText}*/}
+                {/*    addCard={() => this.addCard('', true)}*/}
+                {/*    addCardFromTemplate={this.addCardFromTemplate}*/}
+                {/*    addCardTemplate={this.addCardTemplate}*/}
+                {/*    editCardTemplate={this.editCardTemplate}*/}
+                {/*    readonly={this.props.readonly}*/}
+                {/*/>*/}
```
Comment (hahmadia) [inline webapp/src/components/centerPanel.tsx:141]: Should we just remove this code?

**Review by hahmadia — COMMENTED** [review]
Nice! Thanks Signalwerk! 
How does this behave if the browser is shrunk vertically? I have also one code comment.

File: webapp/src/components/centerPanel.tsx (hunk 1 above)
Comment (harshilsharma63) [reply webapp/src/components/centerPanel.tsx:141]: My bad. Removed now.

**Comment by harshilsharma63** [comment]
> How does this behave if the browser is shrunk vertically?

It works perfectly fine unless someone reduces their browser height to around 250px.

**Review by hahmadia — APPROVED** [review]

**Pull request merged**